chain-signatures/node/src/protocol/mod.rs

```diff
@@ -241,6 +237,22 @@ impl MpcSignProtocol {
                 };
                 tracing::debug!(?contract_state);
 
+                // Sets the latest configurations from the contract:
+                self.ctx.cfg = match rpc_client::fetch_mpc_config(
+                    &self.ctx.rpc_client,
+                    &self.ctx.mpc_contract_id,
+                    &self.ctx.cfg,
+                )
+                .await
+                {
+                    Ok(config) => config,
+                    Err(e) => {
+                        tracing::error!("could not fetch contract's config: {e}");
+                        tokio::time::sleep(Duration::from_secs(1)).await;
+                        continue;
```

Comment:
Is one second too frequent, config is not very frequently changed. 

And above `rpc_client::fetch_mpc_config.await` blocks the main tokio thread, slow near rpc could slowdown protocol state handle(). I would suggest merge config with state call, doing it less frequently, or fetch it in a background tokio thread, without `await`, and `run` read from the fetched config written by the background thread, it could be 1s old but next MpcSign will get it

Reply:
yeah 1s might be too frequent actually for configs that rarely if ever change. We can probably have a background task that fetches the config every 10 minutes or so as to not block this thread